src/DI/ContainerBuilder.php

```diff
@@ -416,6 +416,8 @@ private function resolveImplement(ServiceDefinition $def, $name)
 						throw new ServiceCreationException("Type hint for \${$param->getName()} in $interface::$methodName() doesn't match type hint in $class constructor.");
 					}
 					$def->getFactory()->arguments[$arg->getPosition()] = self::literal('$' . $arg->getName());
+				} elseif (!$def->getSetup()) {
```

Comment:
I will try to implement "Did you mean" feature.
This would be ok?
`Unused parameter $baz when implementing method Bad4::create(). Typo in Bad4::create() parameters? Did you mean $bar?`